tests/Microsoft.DotNet.Framework.Docker.Tests/SdkImageTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class SdkImageTests
```

Comment:
One thing I find that is a little deceiving is that this class doesn't contain all of the SDK tests.  The primary coverage is currently handled in the RuntimeImageTests.  I think this could be a source of confusions for those who are not intimately familiar with the repo.  A comment might be sufficient to address this.  I'm not sure if the RuntimeImageTests could be renamed to help call this out.

Reply:
Renamed `RuntimeImageTests` to `RuntimeSdkImageTests`.
Renamed `SdkImageTests` to `SdkOnlyImageTests`.